### PR TITLE
Adding redundant queue on celery nodes

### PR DESCRIPTION
**Pull request by rohit-dimagi**
##### SUMMARY

* Adding redundant queue on celery nodes.

##### ENVIRONMENTS AFFECTED

ICDS
##### ISSUE TYPE

- Change Pull Request

### CODE REVIEW

**Comment by rohit-dimagi** [comment]
Closing in favor of https://github.com/dimagi/commcare-cloud/pull/3210

File: environments/icds-cas/app-processes.yml [hunk 1]
```diff
@@ -439,56 +217,9 @@ celery_processes:
...
-    ucr_indicator_queue:
-      concurrency: 4
-  'web38':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web39':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web40':
+    icds_aggregation_queue:
```
Comment (snopoke) [inline environments/icds-cas/app-processes.yml]: @calellowitz is it OK to increase the number of process for this queue?

Comment (calellowitz) [reply environments/icds-cas/app-processes.yml]: This queue won't be needed after today, so I don't think this change is necessary, although i'm not sure its harmful per se. I definitely dont think we need to add it to two more machines at this level

Comment (rohit-dimagi) [reply environments/icds-cas/app-processes.yml]: Changed the aggregation queue to have it on only single Server.

**Comment by rohit-dimagi** [comment]
Increased reminder_queue process

**Pull request merged**